frontend/front/src/pages/Admin/home/SurveyVisitProfile.js

```diff
@@ -14,18 +14,29 @@ import { SurveyError } from "../survey/SurveyError";
 import { formatISODate } from "../../../components/DateUtils";
 import { houseToString } from "../../../components/AddressUtils";
 import { withAdminPrefix, ADMIN_SURVEY } from "../../../routing/routes";
+import { buildDataFromSurveyAnswers } from "../../../util/surveyUtils";
 
 const SurveyProfile = () => {
   const navigate = useNavigate();
   const { uid: surveyVisitId } = useParams();
-
   const { data: surveyVisit, error: surveyVisitError } =
     useGetSurveyVisitQuery(surveyVisitId);
 
   const { data: houseData, error: houseError } = useGetHomeQuery(
-    surveyVisit?.homeId,
+    surveyVisit?.home_id,
     { skip: !surveyVisit }
   );
+
+  const surveyAnswers = useMemo(
+    () =>
+      surveyVisit?.survey_response
```

Comment:
You can probably make this `surveyVisit?.survey_response?.survey_answers` like you have it on the next line, but you did do this correctly :+1:

Reply:
Sorry, I don't understand. If you mean the way it is formatted, prettier is formatting it that way.

Reply:
The formatting's fine, I mean you're checking that `surveyVisit?.survey_response` exists in the conditional, but you can make it like an ounce more efficient by changing the check to `surveyVisit?.survey_response?.survey_answers`, to avoid the call to buildDataFromSurveyAnswers if `survey_answers` is null or something. But it's fine as is!